TrkDiag/src/BkgDiag_module.cc

```diff
@@ -29,23 +29,39 @@
 #include "Offline/MCDataProducts/inc/SimParticle.hh"
 #include "Offline/DataProducts/inc/PDGCode.hh"
 #include "Offline/TrkDiag/inc/BkgHitInfo.hh"
-/// Utilities
-#include "Offline/Mu2eUtilities/inc/SimParticleTimeOffset.hh"
-// diagnostics
-#include "Offline/TrkDiag/inc/StrawHitInfo.hh"
+// art
+#include "canvas/Utilities/InputTag.h"
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
 using namespace std;
```

Comment:
you would need to add a few std::string here and there and you could get rid of "using namespace std", which I believe is always a good thing. Or just use using std::string

Reply:
I removed the general 'using namespace' call, I agree that's ugly.